hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -56,10 +60,10 @@
  */
 public class Token {
```

Comment:
A lot of the new operations are not covered by unit tests

Reply:
Actually, i lost the old coverage I provided while merging. I;ve managed to get it back to 94% line on this branch